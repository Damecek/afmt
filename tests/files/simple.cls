public with sharing     class        MyClass {
    Private String s;
        Private String             k;
}
