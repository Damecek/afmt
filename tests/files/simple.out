public with sharing MyClass {
  Private String s;
  Private String k;
}

